block-storage/src/main/scala/coop/rchain/blockstorage/ApprovedStoreSyntax.scala

```diff
@@ -0,0 +1,26 @@
+package coop.rchain.blockstorage
+
+import cats.effect.Sync
+import cats.implicits._
```

Comment:
recommend `cats.syntax.all._`instead of `cats.implicits._`

reason https://discord.com/channels/391435113415573504/410182072054054912/714082912181551104

Reply:
Yes, reasonable. [Fixed](https://github.com/rchain/rchain/pull/3655/commits/61a9b18a9873ce548ae9b3f0c33a2ac488071fe8).